frontend/src/app/signUp/IdentityVerification/PersonalDetailsForm.tsx

```diff
@@ -228,16 +232,50 @@ const PersonalDetailsForm = ({
             </p>
             <h2 className="questions-form-name">{getPersonFullName()}</h2>
           </div>
-          {Object.entries(personalDetailsFields).map(
-            ([key, { label, required, hintText }]) => {
-              const field = key as keyof IdentityVerificationRequest;
-              return (
-                <div key={field}>
-                  {getFormElement(field, label, required, hintText)}
-                </div>
-              );
-            }
+          {getFormElement("dateOfBirth", "Date of birth", true, "", "")}
+          {getFormElement(
+            "preheader1",
+            "Personal contact information",
+            false,
+            "",
+            "personal-contact-group-header"
```

Comment:
non-blocking: I think it could be nice to have these header ids in a variable to pass them along as the id and aria-labelledby value

Reply:
Good suggestion, done!